packages/hoprd/src/index.ts

```diff
@@ -504,7 +511,6 @@ async function main() {
         // Wait for actions to take place
         setTimeout(1e3)
```

Comment:
I noticed this `setTimeout` call might be `awaited` ? cc @nionis

Reply:
yeah it looks like prior to the API v1 removal it was await so we should probably await it again, `--run` has been removed in master 🙏